src/Console/ConfigurationResolver.php

```diff
@@ -588,6 +588,17 @@ private function parseRules()
             return $this->getConfig()->getRules();
         }
 
+        $rules = trim($this->options['rules']);
+
+        if ($rules[0] === '{') {
+            $rules = json_decode($rules, true);
+            if ($error = json_last_error()) {
```

Comment:
Although I like it, please don't assign inside the condition.
And please use `sprintf` to construct the exception message.